Britannia/events/BTWK_storyline_events.txt

```diff
@@ -0,0 +1,386 @@
+# BTWK_story_event.0005 = Uther is Dead global ping
+# BTWK_story_event.0006 = Unique event for player controlled mordred allowing tag switch to arthur
+# BTWK_story_event.0007 = Arthur introduction event
+
+### ARTHUR REGENCY CHAIN ###
+# BTWK_story_event.0008 = Regent decides to give power to Arthur
+# BTWK_story_event.0009 = Arthur accepts regency
+# BTWK_story_event.0010 = Notify Mordred
+# BTWK_story_event.0011 = Notify other vassals
+# BTWK_story_event.0012 = Set Flag
+
+
+############## UTHER'S DEATH and ARTHUR'S RETURN ##############
+
+
+BTWK_story_event.0001 = {  # Triggered by on_game_start on_action incase Uther's pneumonia is cured. Possibly make it non-hidden also. "Second bout of illness..."
+	title = BTWK_story_event.0001.t
+	desc = BTWK_story_event.0001.desc
+	# More fitting music cue/sounds needed
+	theme = skull
+
+	right_portrait = {
+		character = character:1
+		animation = worry
+	}
+	
+	option = {
+		name = BTWK_story_event.0001.a
+		trigger = {
+			NOT = { has_character_flag = utherillness }
+			is_alive = yes	
+		}
+		character:1 = {
+			add_trait = pneumonic
+			add_character_flag = utherillness
+			trigger_event = {
+				id = BTWK_story_event.0002
+				days = {130 180}
+			}
+		}
+		
+	}
+}
+
+BTWK_story_event.0002 = {  # Should probably change this to make it non-hidden & a unique end of life event for Uther.
+	type = character_event
+	title = BTWK_story_event.0002.t
+	desc = BTWK_story_event.0002.desc
+	# More fitting music cue/sounds needed
+	theme = skull
+
+
+	right_portrait = {
+		character = character:1
+		animation = worry
+	}
+	
+	option = {
+		name = BTWK_story_event.0002.a
+		trigger = {
+			this = character:1	
+			has_character_flag = utherillness
+			is_alive = yes
+			NOT = { has_character_flag = utherincapable }
+		}
+		character:1 = {
+			add_trait = incapable
+			add_character_flag = utherincapable
+		}
+		
+	}
+}
+
+BTWK_story_event.0003 = {  #Sub-event that delays Arthur's return, & fires "Uther is dead" event.
+
+	type = character_event
+	hidden = yes
+	
+	trigger = {
+
+		character:1 = {
+			is_alive = no
+		}
+		global_var:uther_is_dead = 0
+	}
+	
+	immediate = {
+
+		change_global_variable = {
+			name = uther_is_dead
+			add = 1
+		}
+
+		#  Fires Arthur's return HIDDEN EVENT  #
+		trigger_event = {
+			id = BTWK_story_event_0004
+		}
+	}
+}
+
+BTWK_story_event_0004 = {
+
+	type = character_event
+	hidden = yes
+
+	immediate = {
+
+		#  Arthur gets d_durnac & vassals  #
+		create_title_and_vassal_change = {
+			type = granted
+			save_scope_as = arthur_pendragon_durnac
+			add_claim_on_loss = yes
+		}
+		title:d_centdumnonia = {
+			every_de_jure_county = {
+				limit = {
+					holder = character:5
+				}
+				change_title_holder = {
+					holder = character:3
+					change = scope:arthur_pendragon_durnac
+				}
+			}
+			change_title_holder = {
+				holder = character:3
+				change = scope:arthur_pendragon_durnac
+			}
+		}
+		title:c_vindocladia.holder = {
```

Comment:
Instead of specifically scoping to c_vindocladia, you should run this for all de jure vassals of d_centdumnonia (at least those that should be vassalized)